cron/poll_pull_requests.py

```diff
@@ -118,10 +119,18 @@ def poll_pull_requests():
                     old_votes[user] = 1
             json.dump(old_votes, fp)
 
+            # flush all buffers because we might restart, which could cause a crash
+            os.fsync(fp)
```

Comment:
@mark-i-m sorry I made a vague comment and then had to leave :)  Don't worry about it...I saw you were looking at making the json writing a little more resilient to restarts, and it reminded me that we need to make this part write to a backup file, then atomic move to the correct file, like in the code I linked, so if we restart or crash during `json.dumps`, our json file won't be corrupted.

I realize that this is probably outside the scope of the PR so don't worry about it, I can take a look at it :)